acc-testcases/data-sources/load_balancer_pool.yaml

```diff
@@ -3,4 +3,4 @@ acc:
     name = "tf_POOL_DO_NOT_DELETE"
     lb_id = 306
   validations:
-    json.loadBalancerPool.id: 1019
+    json.loadBalancerPool.id: 1037
```

Comment:
Why we are hardcoding the ID here? It means we cannot use this acceptance test in other environment correct?

Reply:
yes, we already created these pools for data source to check it and @manjunath-batakurki  also aware about this.

Reply:
Ok approving it now. But check the possibility of acceptance test in this order.

Tier1 gateway-> LB Creation with new tier1 gateway-> LB monitor-> LB pool-> attached-LB-Pool-group->LB virtual server

So at the time of Virtual server creation we can make use the pool ID as reference instead of hardcoding.

CC: @manjunath-batakurki

Reply:
@jinukp06  we already discussed possibility of acceptance test in this order.

Tier1 gateway-> LB Creation with new tier1 gateway-> LB monitor-> LB pool-> attached-LB-Pool-group->LB virtual server

but for the format for current repo will support this way, so for now we created some of them to make useful for the data source.